### DISCUSSION TITLE
Rename Doors?

### DISCUSSION THREAD

**Discussion by S-Mario987** [thoukydides/homebridge-homeconnect#197] [Q&A] [answered]
Hi everyone,

just a quick question. I just updated to the latest version 0.36 and I saw in the Changelog that now the doors are separately.

I just had a Dishwasher, Oven, Refrigerator etc. and I would love to rename the Doors. Now it’s called only „Room Door: open“ but not which device it is. :(.

in the past it was: „Room Devicename“ open\closed

thanks for your help 😊

**Comment by thoukydides** [comment, marked as answer]
Duplicate of #196
